clients/drcachesim/reader/reader.h

```diff
@@ -143,7 +143,9 @@ class reader_t : public std::iterator<std::input_iterator_tag, memref_t> {
     addr_t prev_instr_addr_ = 0;
     int bundle_idx_ = 0;
     std::unordered_map<memref_tid_t, memref_pid_t> tid2pid_;
-    uint64_t last_timestamp_ = 0;
+    uint64_t cur_instr_count_ = 0;
+    uint64_t chunk_instr_count_ = 0;
```

Comment:
As this is expected to be a constant once assigned, I wonder if we should name it as kChunkInstrCount or something.

Reply:
That naming convention is not in place anywhere in dr$sim or DR in general: all-caps normally indicates a constant, but that doesn't feel right here.  I'll just add a comment.